docs/source/glossary.md

```diff
@@ -17,4 +17,20 @@ Functional Programming
   This contrasts with functions or methods that depend on variables that are not explicitly passed as an input (such as accessing `self.variable` inside a method) or that alter the inputs or other state variables in-place, instead of returning new distinct variables as outputs.
 Dispatching
   Choosing which function or method implementation to use based on the type of the input variables (usually just the first variable). For some examples, see Python's documentation for the [singledispatch](https://docs.python.org/3/library/functools.html#functools.singledispatch) decorator.
-:::::
+
+[Equidispersion](http://www.ce.memphis.edu/7012/L20_CountDataModels_v2.pdf)
+  If in a Poisson distribution if the variance equals the mean of the distribution, it is reffered to as equidispersion.
+
+[Generalized Poisson PMF](https://www.sciencedirect.com/science/article/pii/S0047259X14000256)
+  A generalization of the {term}`Poisson distribution`, with two parameters X1, and X2, is obtained as a limiting form of the {term}`generalized negative binomial distribution`. The variance of the distribution is greater than, equal to or smaller than the mean according. as X2 is positive, zero or negative. For formula and more detail visit the link 
```

Comment:
Hi Guneet! Thank you for all the great work you're doing, this PR is looking good! Before merging, let's take a look at the definitions. This fragment needs a bit more work: 

>  The variance of the distribution is greater than, equal to or smaller than the mean according. as X2 is positive, zero or negative.

I think the link provided talks about a more specific case, not the Generalized Poisson Distribution we intend to define. I would use this link https://www.jstor.org/stable/1267389 (which seems to be the source of the definition you used for this term, I would perhaps mention it since this is the original paper that proposed the formula) and maybe also refer to this one in case it's helpful https://arxiv.org/pdf/math/0606238.pdf 

Minor note:
I would also omit this phrase (personal opinion) or, if we leave it, refer to "the link" as "the link in the title" to be more specific.
> For formula and more detail visit the link